packages/components/src/notice/style.scss

```
@@ -36,20 +36,17 @@
 .components-notice__actions {
 	display: flex;
 	flex-wrap: wrap;
+	gap: $grid-unit-15;
 }
 
 .components-notice__action.components-button {
-	&,
-	&.is-link {
-		margin-left: $grid-unit-15;
-	}
 	&.is-secondary {
 		vertical-align: initial;
 	}
 
 	// When it has better support, this can be replaced
 	// with column-gap since these are flex items.
-	margin-right: $grid-unit-10;
+	margin-top: $grid-unit-10;
```

Referencing [the Figma designs for this component](https://www.figma.com/design/804HN2REV2iap2ytjRQ055/WordPress-Design-System?node-id=16432-131127&t=yjqzIJbdM70RINPy-0), I think this margin should be `$grid-unit-15` (12px) instead of `$grid-unit-10` (8px), consistent with the gap between each action button.


```suggestion
	margin-top: $grid-unit-15;
```